### ISSUE TITLE
notification preview throws npe

### ISSUE THREAD

**Issue by farukcankaya** [intercom/intercom-android#203]
## Version info
  - Intercom for Android version: 3.0.0-beta3
  - Android OS version: 4.4.4
  - Android Device: SM G360H

## Logs
``````
Fatal Exception: java.lang.NullPointerException
       at io.intercom.android.sdk.preview.Overlay.moveForward(Overlay.java:238)
       at io.intercom.android.sdk.preview.NotificationManager.onSnippetDismiss(NotificationManager.java:133)
       at io.intercom.android.sdk.preview.Overlay$2.onSpringAtRest(Overlay.java:140)
       at io.intercom.com.facebook.rebound.BaseSpringSystem.io.intercom.com.facebook.rebound.Spring.advance(BaseSpringSystem.java:4445)
       at io.intercom.com.facebook.rebound.AndroidSpringLooperFactory$ChoreographerAndroidSpringLooper$1.doFrame(AndroidSpringLooperFactory.java:118)
       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:759)
       at android.view.Choreographer.doCallbacks(Choreographer.java:574)
       at android.view.Choreographer.doFrame(Choreographer.java:543)
       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:747)
       at android.os.Handler.handleCallback(Handler.java:733)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5433)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1268)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1084)
       at dalvik.system.NativeStart.main(NativeStart.java)
``````

**Comment by ronocod** [comment]
Hi @farukcankaya,

Thanks for reporting this. We'll have a fix for this in the next release.

**Comment by ronocod** [comment]
Fixed in `3.0.0-beta4`.
